examples/Cart/CartAddLineItemAction.json

```diff
@@ -1,5 +1,34 @@
 {
   "action": "addLineItem",
   "productId": "{{productId}}",
-  "variantId": 1
+  "variantId": 2,
+  "quantity": 1,
+  "supplyChannel": {
+    "typeId": "channel",
+    "id": "{{channelId}}"
+  },
+  "distributionChannel": {
+    "typeId": "channel",
+    "id": "{{channelId}}"
+  },
+  "externalTaxRate": {
```

Comment:
Do you think it's a good to show as default an example with external pricing?

Reply:
Wanted to show an example for an externalPrice.
But you're right, I removed it.